app_dart/lib/src/request_handlers/push_benchmark_to_center.dart

```diff
@@ -0,0 +1,124 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+import 'dart:async';
+
+import 'package:appengine/appengine.dart';
+import 'package:gcloud/db.dart';
+import 'package:meta/meta.dart';
+import 'package:metrics_center/flutter.dart' as mc;
+
+import '../datastore/cocoon_config.dart';
+import '../foundation/providers.dart';
+import '../model/appengine/time_series.dart';
+import '../model/appengine/time_series_value.dart';
+import '../request_handling/body.dart';
+import '../request_handling/request_handler.dart';
+import '../service/datastore.dart';
+
```

Comment:
Don't we have a handler to push metrics? can we reuse the existing one instead of creating a new ad hoc handler?